robot/server/server.go

```diff
@@ -56,6 +60,49 @@ func New(robot robot.Robot) pb.RobotServiceServer {
 func (s *Server) Close() {
 }
 
+// Tunnel tunnels traffic to/from the client from/to a specified port on the server.
+func (s *Server) Tunnel(srv pb.RobotService_TunnelServer) error {
+	req, err := srv.Recv()
+	if err != nil {
+		return fmt.Errorf("failed to receive first message from stream: %w", err)
+	}
+
+	dest := strconv.Itoa(int(req.DestinationPort))
```

Comment:
Without finding the proto for this, I'm guessing `DestinationPort` is typed as an int32? Or is this a uint thing?

Reply:
it is a uint32